document/validation/conventions.rst

```diff
@@ -0,0 +1,177 @@
+.. index:: ! validation, ! type system, function type, table type, memory type, globaltype, valtype, resulttype, index space
+
+Conventions
+-----------
+
+Validation checks that a WebAssembly module is well-formed.
+Only valid modules can be :ref:`instantiated <instantiation>`.
+
+Validity is defined by a *type system* over the :ref:`abstract syntax <syntax>` of both instructions and modules.
+For each piece of abstract syntax, there is a typing rule that specifies the constraints that apply to it.
+All rules are given in two *equivalent* forms:
+
+1. In *prose*, describing the meaning in intuitive form.
+2. In *formal notation*, describing the rule in mathematical form.
```

Comment:
When we discussed this, we agreed that formal notation would be placed in an appendix, rather than being used within the main document itself.